spec/_attachments/interface-spec-changelog.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog {#changelog}
 
 ### ∞ (unreleased)
+- Set the maximum depth of a delegation in a read_state response/certified variable certificate to 1.
```

Comment:
I always append the new entry to the bottom of the list. Could you please move it so that we're consistent?

Reply:
Done